### PR TITLE
Improved useFieldArray test

### PR DESCRIPTION
**Pull request by keiya01**

### CODE REVIEW

File: src/useFieldArray.test.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ import {
 } from '@testing-library/react';
 import { act, renderHook } from '@testing-library/react-hooks';
 import { useFieldArray } from './useFieldArray';
-import { reconfigureControl } from './__mocks__/reconfigureControl';
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:11]: wohooooooo 🙏🏻

File: src/useFieldArray.test.tsx [hunk 2]
```diff
@@ -637,6 +636,41 @@ describe('useFieldArray', () => {
...
+
+        return (
+          <form>
+            {fields.map((field, i) => (
+              <input
+                key={field.id}
+                type="text"
+                name={`test[${i}].value`}
+                ref={register()}
+                defaultValue={field.value}
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:654]: i reckon we can have dev warning here as well for users who is using useFieldArray and missing defaultValue. i will be looking into that 👍

**Review by bluebill1049 — APPROVED** [review]
TOO GOOD! Thank you @keiya01

**Comment by keiya01** [comment]
Thank you for your review 🙏

**Pull request merged**